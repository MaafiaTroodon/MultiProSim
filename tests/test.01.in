2 5 1
Proc1 3 1 1
SEND 102
RECV 102
HALT

Proc2 3 1 1
RECV 101
SEND 101
HALT
