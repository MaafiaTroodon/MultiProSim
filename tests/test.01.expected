[01] 00000: process 1 new
[01] 00000: process 1 ready
[01] 00000: process 1 running
[01] 00000: process 2 new
[01] 00000: process 2 ready
[01] 00001: process 1 blocked (send)
[01] 00001: process 2 running
[01] 00002: process 2 blocked (recv)
[01] 00003: process 1 ready
[01] 00003: process 1 running
[01] 00003: process 2 ready
[01] 00004: process 1 blocked (recv)
[01] 00004: process 2 running
[01] 00005: process 2 blocked (send)
[01] 00006: process 1 finished
[01] 00006: process 2 finished
| 00006 | Proc 01.01 | Run 2, Block 0, Wait 0, Sends 1, Recvs 1
| 00006 | Proc 01.02 | Run 2, Block 0, Wait 2, Sends 1, Recvs 1
